transfers/transfer.py

```diff
@@ -296,7 +300,7 @@ def start_transfer(ss_url, ts_location_uuid, ts_path, depth, am_url, user_name,
         return None
     if not response.ok or resp_json.get('error'):
         LOGGER.error('Unable to start transfer.')
-        LOGGER.debug('Response: %s', resp_json)
+        LOGGER.error('Response: %s', resp_json)
```

Comment:
It was, but any time you see that message you have an error, and it's always useful to be able to see the erroneous response (and not have to look for DEBUG logging)